packages/smooth_app/lib/pages/product/add_basic_details_page.dart

```diff
@@ -168,39 +166,23 @@ class _AddBasicDetailsPageState extends State<AddBasicDetailsPage> {
         ),
       );
 
-  Future<bool> _saveData(LocalDatabase localDatabase) async {
+  Future<bool> _saveData(
+      LocalDatabase localDatabase, Product inputProduct) async {
```

Comment:
First thing first, please add a ',' after `inputProduct`.

Then, I would not recommend using the `inputProduct`, because you're changing it. You should start from scratch with a new `Product()`.

Reply:
Thanks for your suggestion, I removed the parameter inputProduct